src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1668,26 +1668,50 @@ private DiagnosticInfo GetBadEventUsageDiagnosticInfo(EventSymbol eventSymbol)
                 new CSDiagnosticInfo(ErrorCode.ERR_BadEventUsageNoField, leastOverridden);
         }
 
-        internal static bool AccessingAutoPropertyFromConstructor(BoundPropertyAccess propertyAccess, Symbol fromMember)
+        internal static bool IsReadingThroughBackingFieldInConstructor(BoundPropertyAccess propertyAccess, Symbol fromMember)
```

Comment:
>IsReadingThroughBackingFieldInConstructor [](http://example.com/codeflow?start=29&length=41)

It looks like there is only one call site for this helper. Is there much benefit to have this separate helper? Could we use ```IsPropertyAssignedThroughBackingField``` instead for now?

Reply:
> for now

If we're going to need it later, then I personally prefer keeping it in place. If you think this will need adjustments, we can have PROTOTYPE comments for that.

Reply:
> If we're going to need it later, then I personally prefer keeping it in place.

That would be fine.

> If you think this will need adjustments, we can have PROTOTYPE comments for that.

I guess it would be good to have a comment to come back and reevaluate if a separate helper is still needed.